dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -606,6 +612,7 @@
         this.executorService = executorService;
         this.threadPool = pool;
         this.coreThreads = pool.getCorePoolSize();
+        this.currentMinimumPoolSize = this.coreThreads;
```

Comment:
currentMinimumPoolSize is not reset in resetThreadPool.  Should it be?  Or should resetThreadPool use currentMinimumPoolSize instead of coreThreads in its calculation.

Reply:
resetThreadPool zeros out all the stats - so on that basis, I'm going with setting currentMinimumPoolSize to coreThreads 
good catch :)